marketplace_docs/templates/Fabric/interactive_script.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+clear
+
+cat <<EOM
+
+Zulip will ask you for the following two input for completing the installation:
+
+- The email address of the person or team who should get support and error emails from this Zulip server.
+- The user-accessible domain name for this Zulip server, i.e., what users will type in their web browser.
+
+To skip the Zulip setup for now, press Ctrl+C.  You will be prompted again on your next login.
+
+You can also re-run this setup script at any time with the command:
+   bash $(readlink -f "${0}")
+
+Please press enter when you are ready to configure Zulip.
+
+EOM
+
+read -r _
+
+while  [ -z "${email}" ] && [ -z "${hostname}" ]; do
+read -r -p "Enter the email address for recieving support and error emails (ex. user@example.com) " email
```

Comment:
Email address for support and error emails (e.g. ops@example.com):

Reply:
Maybe "Administrator email address to receive support and error emails"?

We might also want to put a link to this in the content above:

https://zulip.readthedocs.io/en/latest/production/install.html#installer-options